script.js

```diff
@@ -0,0 +1,39 @@
+function calculate() {
+    var operationDropDown = document.getElementById("operation")
+    var firstNumElement = document.getElementById("firstNumber")
+    var secondNumElement = document.getElementById("secondNumber")
+    var firstNumber = parseInt(firstNumElement.value)
+    var secondNumber = parseInt(secondNumElement.value)
```

Comment:
These would be ok to do as one liners since you are only creating the element variables to then get their value. If writing it like this helps understand it always start there but once you are done and everything works this would be a good thing to look at for potential clean up.

```JavaScript
var firstNumber = parseInt(document.getElementById("firstNumber").value)
var secondNumber = parseInt(document.getElementById("secondNumber").value)
```